janus-frontend/src/services/http/index.ts

```diff
@@ -4,8 +4,9 @@ const getHandler = (url: string) => {
 	return fetch(`${BASE_URL}${url}`);
 };
 
-const postHandler = (url: string, payload: Record<string, string>) => {
+const postHandler = (url: string, payload: any) => {
 	return fetch(`${BASE_URL}${url}`, {
+		// url = /link
```

Comment:
remove comment

Reply:
I've kept it for the reference for a later time